views/presenter.html

```diff
@@ -44,8 +45,8 @@ <h1 class="govuk-heading-xl">Who can we contact about this application?</h1>
 
         {{ govukInsetText({
           html: '
-            <h2 class="govuk-heading-m">Information shown on the public register</h2>
-            <p>None of the information provided in this section will be shown on the public Register of Overseas Entities.</p>
+            <h2 class="govuk-heading-m">What information we’ll show on the public register</h2>
```

Comment:
Why you do not want to use the `inset-text` as [here](https://github.com/companieshouse/overseas-entities-web/blob/main/views/includes/inset-text.html)?

Reply:
Inset text will add a bar to the left hand side, which is not what has been requested. Also this is more consistent with top page hint text for other pages such as due diligence.

Reply:
Could you please check the screenshot on your story. Anyway they should be all in the same style, so eventually check with UX and confirm that all the inset text related to the information shown on the public register should have all the same style.

Reply:
Sorry thought you were talking about something else , yes we can use inset text here.

Reply:
Updated this.